contributors/devel/welcome-to-kubernetes-new-developer-guide.md

```diff
@@ -0,0 +1,128 @@
+
+2017-06-21 through 2017-08-06
+Original author: [Alexander Campbell](https://github.com/alexandercampbell)
+
+---
+
+_This document assumes that you know what Kubernetes does. If you don't, search
+YouTube for ["Kubernetes"](https://www.youtube.com/watch?v=8SvQqZNP6uo) and try
+the demo at [https://k8s.io/](https://k8s.io/)._
+
+Introduction
+------------
+
+Have you ever wanted to contribute to the coolest cloud technology? This
+document will help you understand the organization of the Kubernetes project and
+direct you to the best places to get started. By the end of this doc, you'll be
+able to pick up issues, write code to fix them, and get your work reviewed and
+merged. If you have questions that aren't answered in this document, feel free
+to suggest changes to it directly or email me at
+[alexanderhcampbell@gmail.com](mailto:alexanderhcampbell@gmail.com).
+
+Special Interest Groups
+-----------------------
+
+Kubernetes developers work in teams called Special Interest Groups (SIGs). At
+the time of this writing there are 31 SIGs (I got the number 31 by going to the
+kubernetes/community repository and running `ls | rg '^sig-' | wc -l`).
+
+The developers within each SIG have autonomy and ownership over that SIG's part
+of Kubernetes. SIGs organize themselves by submitting markdown design documents
+to the [kubernetes/community](https://github.com/kubernetes/community)
+repository. You can see an example of this planning process at
+[kubernetes/community#645](https://github.com/kubernetes/community/pull/645). Like everything else in Kubernetes, a SIG is an open,
+community, effort. In each SIG, full-time employees from the
+Kubernetes Partners work side-by-side with volunteer contributors. Anybody is
+welcome to jump into a SIG and begin fixing issues, critiquing design proposals
+and reviewing code.
+
+Most people who visit the Kubernetes repository for the first time are
+bewildered by the thousands of open issues in our main repository. But now that
+you know about SIGs, it's easy to filter by labels to see what's going on in
+your SIG.
+
+Downloading, Building, and Testing Kubernetes
+---------------------------------------------
+
+This guide is non-technical, so I'm going to refer you to our online
+documentation.We have plenty of documentation available under
+[github.com/kubernetes/community/contributors/devel](https://github.com/kubernetes/community/tree/master/contributors/devel).
+Check out
+[development.md](https://github.com/kubernetes/community/blob/master/contributors/devel/development.md)
+for more details.
+
+Pull-Request Process
+--------------------
+
+When you submit a pull request to
+[kubernetes/kubernetes](https://github.com/kubernetes/kubernetes), the Three
+Kubernetes Bots will show up and start applying labels and assigning reviewers.
+Our bots live in
+[github.com/kubernetes/test-infra](https://github.com/kubernetes/test-infra) and
+their names are:
+
+ 1. @k8s-bot
+ 2. @k8s-ci-robot
+ 3. @k8s-merge-robot
+
```

Comment:
:) actually me neither and that's why I thought about it. As I look at various PRs and Issues - I only notice @k8s-ci-robot and @k8s-merge-robot in play. Hopefully some reviewer will provide some info, if not I wonder if we should remove it from the list. The https://github.com/kubernetes/test-infra/blob/master/commands.md#k8s-bot-commands provides commands for ci-robot and merge-robot.